homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -343,23 +356,29 @@ async def _async_replace_device(self, replace_device):
 
         device_registry.async_remove_device(old_device)
 
-    def _can_add_device(self, new_rfx_obj):
+    def _can_add_device(self, new_rfx_obj: rfxtrxmod.RFXtrxEvent) -> bool:
         """Check if device does not already exist."""
         new_device_id = get_device_id(new_rfx_obj.device)
         for packet_id, entity_info in self._config_entry.data[CONF_DEVICES].items():
             rfx_obj = get_rfx_object(packet_id)
+            assert rfx_obj
+
             device_id = get_device_id(rfx_obj.device, entity_info.get(CONF_DATA_BITS))
             if new_device_id == device_id:
                 return False
 
         return True
 
-    def _can_replace_device(self, entry_id):
+    def _can_replace_device(self, entry_id: str) -> bool:
         """Check if device can be replaced with selected device."""
+        assert self._selected_device_object
+
         device_data = self._get_device_data(entry_id)
```

Comment:
`_get_device_data` can maybe be moved to a standalone function.

Reply:
Sure, but i'd rather not change logic/structure in a pull to add full typing.